### ISSUE TITLE
Script not responding when using tooltip on a line or scatterline graph in IE 11

### ISSUE THREAD

**Issue by Yves-Be** [telerik/kendo-angular2#163]
Page stops responding when hovering over a marker with IE 11. Works fine in Firefox and Chrome.

```
<kendo-chart>
    <kendo-chart-tooltip></kendo-chart-tooltip>
    <kendo-chart-series>
                <kendo-chart-series-item [data]="[[1,1],[2,2],[3,3],[4,4]]" type="scatterLine">
                    <kendo-chart-series-item-tooltip>
                        <template let-value="value">
                            <p>tooltip {{ value.x }}, {{ value.y }}</p>
                        </template>
                    </kendo-chart-series-item-tooltip>
                </kendo-chart-series-item>
    </kendo-chart-series>
</kendo-chart>
```

http://plnkr.co/edit/BORkA65yOgjRaRQfDHGp?p=preview

**Comment by tsvetomir** [comment]
Thanks for reporting this issue. We've got to the bottom of it and it should be fixed in [v0.14.6](https://github.com/telerik/kendo-angular-popup/releases/tag/v0.14.6) of the Popup.

Note that the site has not been updated to this version yet, but the package is available for download.

**Comment by Yves-Be** [comment]
Can confirm it's fixed, thanks for the quick response.
